contracts/proposals/ccip016-missed-payouts.clar

```diff
@@ -0,0 +1,421 @@
+;; TRAITS
+
+(impl-trait .proposal-trait.proposal-trait)
+(impl-trait .ccip015-trait.ccip015-trait)
+
+;; ERRORS
+
+(define-constant ERR_PANIC (err u16000))
+(define-constant ERR_SAVING_VOTE (err u16001))
+(define-constant ERR_VOTED_ALREADY (err u16002))
+(define-constant ERR_NOTHING_STACKED (err u16003))
+(define-constant ERR_USER_NOT_FOUND (err u16004))
+(define-constant ERR_PROPOSAL_NOT_ACTIVE (err u16005))
+(define-constant ERR_PROPOSAL_STILL_ACTIVE (err u16006))
+(define-constant ERR_VOTE_FAILED (err u16007))
+
+;; CONSTANTS
+
+(define-constant SELF (as-contract tx-sender))
+(define-constant CCIP_016 {
+  name: "Refund Incorrect CCD007 Payouts",
+  link: "https://github.com/citycoins/governance/blob/feat/add-ccip-016/ccips/ccip-016/ccip-016-refund-incorrect-ccd007-payouts.md",
+  hash: "",
+})
+;; set city ID
+(define-constant MIA_ID (default-to u1 (contract-call? .ccd004-city-registry get-city-id "mia")))
+(define-constant NYC_ID (default-to u2 (contract-call? .ccd004-city-registry get-city-id "nyc")))
+
+(define-constant VOTE_SCALE_FACTOR (pow u10 u16)) ;; 16 decimal places
+(define-constant MIA_SCALE_BASE (pow u10 u4)) ;; 4 decimal places
+(define-constant MIA_SCALE_FACTOR u8916) ;; 0.8916 or 89.16%
+
+;; DATA VARS
+
+;; vote block heights
+(define-data-var voteActive bool true)
+(define-data-var voteStart uint u0)
+(define-data-var voteEnd uint u0)
+
+;; start the vote when deployed
+(var-set voteStart block-height)
+;; vote tracking
+(define-data-var yesVotes uint u0)
+(define-data-var yesTotal uint u0)
+(define-data-var noVotes uint u0)
+(define-data-var noTotal uint u0)
+
+;; DATA MAPS
+(define-map CityVotes
+  uint ;; city ID
+  { ;; vote
+    totalAmountYes: uint,
+    totalAmountNo: uint,
+    totalVotesYes: uint,
+    totalVotesNo: uint,
+  }
+)
+(define-map UserVotes
+  uint ;; user ID
+  { ;; vote
+    vote: bool,
+    mia: uint,
+    nyc: uint,
+  }
+)
+
+;; PUBLIC FUNCTIONS
+
+(define-public (execute (sender principal))
+  (begin
+    ;; check vote is complete/passed
+    (try! (is-executable))
+    ;; update vote variables
+    (var-set voteEnd block-height)
```

Comment:
We could set this one to be a true 2100 block / 2 week vote since we're not rushing to have it done by a block height. [CCIP-024 did the same thing](https://explorer.hiro.so/txid/SP8A9HZ3PKST0S42VM9523Z9NV42SZ026V4K39WH.ccip024-miamicoin-signal-vote?chain=mainnet).

If we do that we'd have to switch to tenure or burn block height, so maybe better to leave as-is? Block heights move fast now!